collection-scripts/gather

```diff
@@ -24,13 +25,22 @@ resources+=(clusternetworks hostsubnets)
 # Autoscaler Resources
 resources+=(clusterautoscaler machineautoscaler)
 
+PIDS=()
 # Run the Collection of Resources using must-gather
 for resource in ${resources[@]}; do
-    /usr/bin/openshift-must-gather inspect ${resource}
+    LOG_DIR="${BASE_COLLECTION_PATH}/must-gather-resource-logs"
+    mkdir -p ${LOG_DIR}
+    /usr/bin/openshift-must-gather inspect ${resource} 2> ${LOG_DIR}/${resource/\//_}.log &
```

Comment:
```suggestion
    /usr/bin/openshift-must-gather inspect ${resource} 2> ${LOG_DIR}/${resource//[^A-Za-z0-9._-]/_}.log &
```

Reply:
I don't understand the sugestion, my code converts resources with '/' in them like `ns/default` to resources names with '_' in them like `ns_default`, what is yours doing?

This avoid bash path issues when I go to save the resource.

Reply:
@sanchezl can you followup here?